examples/kernel_shap_wine_lr.ipynb

```diff
@@ -0,0 +1,920 @@
+{
```

Comment:
<ul><li>Maybe rephrase the sentence "RHS is a function of a random variable mean"? It's not quite clear how the expected value and the link function are related</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>